Packs/CohesityHelios/Classifiers/classifier-CohesityHelios.json

```diff
@@ -0,0 +1,20 @@
+{
+    "defaultIncidentType": "Cohesity Helios Ransomware Incident",
+    "description": "",
+    "feed": false,
+    "id": "b28ced57-1175-4ea3-807c-e2ce6c96f6a2",
```

Comment:
Please make sure that the ID and the name are the same

Reply:
Sorry, did not follow: should it be changed to 

"id": "Cohesity Incident Classifier"

Reply:
Thank you. Pushed the change.